processing/src/test/java/io/druid/segment/filter/BoundFilterTest.java

```diff
@@ -77,10 +77,10 @@ public BoundFilterTest(
   public void testLexicographicMatchEverything()
```

Comment:
@gianm Added JavascriptFilterTest, SearchQueryFilterTest, RegexFilterTest in addition to the current set, added a test for extraction functions to each